x/profiles/keeper/msg_server_dtag_transfers.go

```diff
@@ -157,29 +157,26 @@ func (k MsgServer) AcceptDTagTransferRequest(goCtx context.Context, msg *types.M
 			senderAcc = authtypes.NewBaseAccountWithAddress(add)
 		}
 
-		receiverProfile, err = types.NewProfileFromAccount(dTagToTrade, senderAcc, ctx.BlockTime())
+		requestSenderProfile, err = types.NewProfileFromAccount(dTagToTrade, senderAcc, ctx.BlockTime())
 		if err != nil {
 			return nil, err
 		}
 	} else {
-		receiverProfile.DTag = dTagToTrade
+		requestSenderProfile.DTag = dTagToTrade
 	}
 
-	// Validate the receiver profile
-	err = k.ValidateProfile(ctx, receiverProfile)
+	// Validate the request sender profile
+	err = k.ValidateProfile(ctx, requestSenderProfile)
 	if err != nil {
 		return nil, errors.Wrap(sdkerrors.ErrInvalidRequest, err.Error())
 	}
 
-	// Save the receiver profile
-	err = k.Keeper.SaveProfile(ctx, receiverProfile)
+	// Save the request sender profile
+	err = k.Keeper.SaveProfile(ctx, requestSenderProfile)
 	if err != nil {
 		return nil, err
 	}
 
-	k.DeleteAllUserIncomingDTagTransferRequests(ctx, msg.Receiver)
-	k.DeleteAllUserIncomingDTagTransferRequests(ctx, msg.Sender)
```

Comment:
These are duplicate with the `storeProfileWithoutDTagCheck` called by `SaveProfile`:
https://github.com/desmos-labs/desmos/blob/405f77bbe774ff3d2dd264f0f929959ffd1151cf/x/profiles/keeper/keeper.go#L120-L121